index.bs

```diff
@@ -2496,7 +2496,7 @@ and the identity of the calling application.
     :: The version number of Google Play Services responsible for providing the SafetyNet API.
 
     : response
-    :: The value returned by the above SafetyNet API. This value is a JWS [[RFC7515]] object (see
+    :: The UTF8 encoded result of the getJwsResult() call of the SafetyNet API. This value is a JWS [[RFC7515]] object (see
```

Comment:
Use `{{UTF-8 encoded}}` to link to https://encoding.spec.whatwg.org/#utf-8-encode.